sim_benchmark/README.md

```diff
@@ -0,0 +1,48 @@
+# Sim Benchmark
```

Comment:
Just "benchmark" for the directory name.

Reply:
OK, I'll change the name.

Reply:
Done.